### PR TITLE
Fix: Do not show ignore messages for default ignored files (fixes #4931)

### LINKED ISSUE #4931: 2.0.0-beta.1 now warns about files in node_modules
When I ugrade eslint from `2.0.0-alpha-2` to `2.0.0-beta.1`, thousands of these warnings show up:

```
~/projects/my-project/node_modules/ws/index.js
  0:0  warning  File ignored because of a matching ignore pattern. Use --no-ignore to override
```

I'm running eslint with `eslint "**/*.js"` and without any ignores. The documentation states that `node_modules` are ignored by default, which is what I want.

From skimming through the commits, I guess this was introduced in https://github.com/eslint/eslint/pull/3948 but I don't understand enough about the internals to look into this myself.

Thanks for helping out.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/29833413-2-0-0-beta-1-now-warns-about-files-in-node_modules?utm_campaign=plugin&utm_content=tracker%2F282608&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F282608&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by gyandeeps**

### CODE REVIEW

File: lib/ignored-paths.js [hunk 1]
```diff
@@ -146,8 +146,6 @@ function IgnoredPaths(options) {
      * @returns {array} raw ignore rules
      */
     function addIgnoreFile(ig, filepath) {
-        var fileContent = fs.readFileSync(filepath, "utf8");
-        fileContent = fileContent.replace(/\*\*/g, "*");
         return ig.addIgnoreFile(filepath);
     }
```
Comment (gyandeeps) [inline lib/ignored-paths.js]: Found expensive dead code here doing a unnecessary I/O operation. :stuck_out_tongue_closed_eyes:

**Comment by ilyavolodin** [comment]
LGTM

File: lib/ignored-paths.js (hunk 1 above)
Comment (nzakas) [inline lib/ignored-paths.js]: Great victory!

**Comment by nzakas** [comment]
This changes looks good, but I don't think it fixes the issue. The issue was saying that `node_modules` wasn't being ignored.

**Comment by gyandeeps** [comment]
It does get ignored (ie file are not linted) but the warning for that ignore is getting printed.

**Comment by nzakas** [comment]
Ohh, I see. Thanks for clarifying.

**Pull request merged**